sencha-workspace/packages/slate-cbl/src/view/student/skill/OverviewWindow.js

```diff
@@ -0,0 +1,85 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+Ext.define('Slate.cbl.view.student.skill.OverviewWindow', {
+    extend: 'Ext.window.Window',
+    xtype: 'slate-cbl-student-skill-overviewwindow',
+    requires: [
+        'Slate.cbl.view.student.skill.OverviewWindowController',
+        'Slate.cbl.view.skill.OverviewBody',
+        'Slate.cbl.model.Skill',
+
+        'Ext.form.field.ComboBox',
+        'Ext.data.ChainedStore'
+    ],
+
+    controller: 'slate-cbl-student-skill-overviewwindow',
+
+    config: {
+        student: null,
+        competency: null,
+        skill: null
+    },
+
+    title: 'Skill Overview',
+    width: 700,
+    minWidth: 700,
+    // constrainHeader: true,
+    autoScroll: true,
+
+    dockedItems: [
+        {
+            dock: 'top',
+
+            xtype: 'toolbar',
+            items: [
+                'Standard:',
+                {
+                    reference: 'skillCombo',
+                    flex: 1,
```

Comment:
The first one isn't stray, we follow a convention of separating options to the container (`flex` and `reference`) from options to the component (everything else).

Sometimes when there are a lot of config options we further break them by which class in the hierarchy they apply to (i.e. a combobox is also a field and also a component), but the following two line breaks in this case don't make sense